recipes/elektronn/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "elektronn" %}
+{% set version = "1.0.13" %}
+{% set sha256 = "e5ba5dd4c5333576dde668977b9cced446eab5fdd3219d91d75550d1e8d5e2fc" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/e/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [not linux64 or not py27]
+
+requirements:
+  build:
+    - python <3
+    - setuptools
+    - cython
+    - h5py
```

Comment:
Drop all of the listed build requirements after h5py. Except for `gcc`. You technically don't need that one either since our docker image for linux builds has gcc in it, but it's not going to hurt anything to have gcc listed here